tools/serverpod_cli/bin/serverpod_cli.dart

```diff
@@ -115,6 +115,20 @@ Future<void> _main(List<String> args) async {
     help:
         'Template to use when creating a new project, valid options are "server" or "module"',
   );
+  createParser.addOption(
+    'postgresPort',
+    abbr: 'p',
+    defaultsTo: '8090',
```

Comment:
No need for `defaultsTo` if postgresPort is optional in `performCreate`. If using `defaultsTo`, the defined constant should be used.

Reply:
sure